### ISSUE TITLE
Font Decoding Bug/Error

### ISSUE THREAD

**Issue by CarlBeek** [zkparty/trusted-setup-frontend#120]
It appears there is something wrong with the font being imported. In particular, there appears to be a decoding error:

**Firefox console:**
```
downloadable font: rejected by sanitizer (font-family: "text-security-disc" style:normal weight:400 stretch:100 src index:0) source: https://ceremony.ethereum.org/static/media/text-security-disc.9f5e7b99e38f4dc390d9.woff2
```
**Chrome console:**
```
Failed to decode downloaded font: https://ceremony.ethereum.org/static/media/text-security-disc.9f5e7b99e38f4dc390d9.woff2
```

**Comment by NicoSerranoP** [comment]
This is related to #119 right? I just noticed that the console errors were caused by a server configuration (checkout this [answer](https://stackoverflow.com/a/71532487/14515077)).

I just added the woff2 and otf file types and it seems that the console errors dissapeared. I couldn't replicate the bug in #119 but I have the feeling that it might be related to network connection.

To hide the text input and avoid browsers asking to save the field as password, we decided to use a custom font family "text-security-disc". It may take a couple of seconds to change between font families if the connection is slow.
